recipes/sqlframe/recipe.yaml

```diff
@@ -0,0 +1,49 @@
+schema_version: 1
+
+context:
+  name: sqlframe
+  version: 3.28.2
+  python_min: 3.9
```

Comment:
We need to define this only if the package min is higher than conda-forge's min.

```suggestion
```